source/framework/vx_objarray.c

```diff
@@ -240,8 +276,7 @@ VX_API_ENTRY vx_reference VX_API_CALL vxGetObjectArrayItem(
         if ((ownIsValidSpecificReference(vxCastRefFromObjectArray(objarr), (vx_enum)VX_TYPE_OBJECT_ARRAY) ==
                 (vx_bool)vx_true_e) && /* TIOVX-1896- LDRA Uncovered Branch Id: TIOVX_BRANCH_COVERAGE_TIVX_OBJARRAY_UBR001 */
                 (obj_desc != NULL) &&
-            (index < obj_desc->num_items) &&
-            (objarr->base.is_virtual == (vx_bool)vx_false_e))
```

Comment:
Why was this check removed?

Reply:
Per email discussion, this was done to align pyramid and object array behavior.  Therefore, it is a valid change.

Closing the comment accordingly